docs/base.rst

```diff
@@ -0,0 +1,7 @@
+ManticoreBase
+=============
+
+.. autoclass:: manticore.core.manticore.ManticoreBase
+   :members:
+   :special-members: __init__
+   :exclude-members: all_states, ready_states, count_ready_states, count_busy_states, killed_states, count_killed_states, terminated_states, count_terminated_states
```

Comment:
maybe: finalize, locked_context,

Reply:
You want those excluded from the base?

Currently I exclude the ones above because I thought they were more appropriate for the states section. All other documented members for ManticoreBase should be present.

https://manticore.readthedocs.io/en/latest/base.html#manticore.core.manticore.ManticoreBase.finalize

https://manticore.readthedocs.io/en/latest/base.html#manticore.core.manticore.ManticoreBase.locked_context

Reply:
oh exclude/